config/config.go

```diff
@@ -13,6 +13,7 @@ type Config struct {
 	HealthCheckInterval        time.Duration `envconfig:"HEALTHCHECK_INTERVAL"`
 	HealthCheckCriticalTimeout time.Duration `envconfig:"HEALTHCHECK_CRITICAL_TIMEOUT"`
 	AWSRegion                  string        `envconfig:"AWS_REGION"`
+	AWSCognitoUserPoolID       string        `envconfig:"AWS_COGNTIO_USER_POOL_ID"`
```

Comment:
Is this potentially sensitive information? If so should we use the json omit tag so it's not displayed in logging.